nixos/modules/services/misc/dump1090-fa.md

```diff
@@ -0,0 +1,29 @@
+# Dump1090-fa {#module-services-dump1090-fa}
+
+[dump1090-fa](https://github.com/flightaware/dump1090) is a demodulator and decoder for ADS-B, Mode S, and Mode 3A/3C aircraft transponder messages. It can receive and decode these messages from an attached software-defined radio or from data received over a network connection.
+
+## Configuration {#module-services-dump1090-fa-configuration}
+
+When enabled, this module automatically creates a systemd service to start the `dump1090-fa` application. The application will then write its JSON output files to `/run/dump1090-fa`.
+
+Exposing the integrated web interface is left to the user's configuration. Below is a minimal example demonstrating how to serve it using Nginx:
+
+```nix
+{ config, pkgs, ... }: {
+  services.dump1090-fa.enable = true;
+
+  services.nginx = {
+    enable = true;
+    virtualHosts."dump1090-fa" = {
+      listen = [{
+        addr = "127.0.0.1";
+        port = 8080;
+      }];
```

Comment:
```suggestion
```

It'd even keep it more concise, people can check the nginx options if they want a custom bind address / port.

Reply:
Agree, done